assets/js/vue-apps/components/grants-back-to-search.vue

```diff
@@ -0,0 +1,46 @@
+<script>
+import { storageAvailable, getWithExpiry } from '../../helpers/storage';
+import IconArrowLeft from './icon-arrow-left.vue';
+const canStore = storageAvailable('localStorage');
+
+export default {
+    components: { IconArrowLeft },
+    props: ['label', 'storageKey', 'prefix'],
+    data() {
+        return {
+            returnLink: undefined
+        };
+    },
+    computed: {
+        id() {
+            return Math.random()
+                .toString(36)
+                .substr(2, 9);
+        }
+    },
+    mounted: function() {
+        if (canStore) {
+            const searchData = getWithExpiry({ type: 'localStorage', key: this.storageKey });
```

Comment:
As `getWithExpiry` references `storageAvailable` do we need the explicit `canStore` here?